src/data/content/quality.ts

```diff
@@ -5,9 +5,193 @@ const techContent = [
+          },
+        ],
+      },
+      {
+        name: "Wait commands in Selenium",
+        intro:
+          "Wait commands in Selenium enable the QAs to pause the execution of test cases for a specified length time. These commands help in observing and troubleshooting errors that may arise due to time variations or any lags. There are two types of wait: ",
+        data: [
+          {
+            name: "Implicit wait",
+            description: `This command instructs the WebDriver to wait for a specified time before throwing an exception. <ul><li>Refer to the command below:<br>driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);</li><li>It can be defined globally for the driver and it will be set for the life of the WebDriver object instance. The specified time is the maximum time, if the element appears before the max time, it will perform action. Disadvantage is that it may mask performance issues of the application. </li></ul>`,
+          },
+          {
+            name: "Explicit wait commands",
+            description: `These commands instruct the WebDriver to wait until a particular condition occurs before executing further scripts. <ul><li>WebDriverWait wait = new WebDriverWait(driver,30);</li><li>Explicit wait is used to target specific condition and wait until an expected condition is met - the expected condition can be visibility of an element, element to be clickable, frame to be visible etc. with the maximum time limit until which it will wait.</li></ul>`,
+          },
+          {
+            name: "Fluent wait",
+            description: `This is used to define maximum wait time for the driver and poll for the given frequency until the condition is met. It can be configured to ignore specific type of exceptions such as 'NoSuchElementException'.`,
+          },
+        ],
+      },
+      {
+        name: "",
+        intro: `To understand the in-depth implementation of how these commands operate in Selenium, refer to this detailed guide on <a href="https://www.selenium.dev/documentation/webdriver/waits/">wait commands in Selenium WebDriver.</a> `,
+        data: [],
+      },
+      {
+        name: "How to use Selenium WebDriver: Example",
+        intro: `The code below automates the login function on a sample application (example: BrowserStack) Sign-in page in Chrome: `,
+        data: [
+          {
+            name: "",
+            description: ``,
+          },
+        ],
+      },
+    ],
+    examples: [
+      {
+        name: "",
+        imageUrl:
+          "https://3fxtqy18kygf3on3bu39kh93-wpengine.netdna-ssl.com/wp-content/uploads/2021/09/Selenium-WebDriver-for-Automation-Testing@2x.webp",
```

Comment:
This image should be under the Selenium Web Driver Framework Architecture (refer content doc)